.github/workflows/validate.yml

```diff
@@ -35,6 +35,17 @@ jobs:
         with:
           files: components/**/*
 
+  validate-pixel-components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Validate Pixel Components
+        uses: nhalstead/validate-json-action@0.1.3
+        with:
+          schema: /components/pixel/schema.json
+          jsons: components/pixel/*/definition.json
+
```

Comment:
Don't need this anymore, just add `pixel` to the `componentType` list above on line 11